### ISSUE TITLE
Grafana Text Panel not rendering in PDF

### ISSUE THREAD

**Issue by dattasoumya92** [IzakMarais/reporter#244]
Hi, 
I have setup the reporter service as a docker container and trying to generate the pdf report from grafana dashboard but i'm getting timeout error when there is a text panel within a dashboard. 

I'm using grafana v 7.1.1
Below is the log I'm getting when generating the pdf:

error getting panel {Id:21 Type:text Title:Costs (\$) displayed are approximate GridPos:{H:3 W:24 X:0 Y:0}}: Error obtaining render: 500 Internal Server Error

Thanks

**Comment by thibmac** [comment]
Hi, 

had the same issue as you, but this is a problem from Grafana.
Rendering a text panel from there doesn't work either :
Rolling back to 7.0.6 will fix this.

Best

**Comment by dattasoumya92** [comment]
> Hi,
> 
> had the same issue as you, but this is a problem from Grafana.
> Rendering a text panel from there doesn't work either :
> Rolling back to 7.0.6 will fix this.
> 
> Best

hey, 

yes, the problem is with Grafana not the reporter, I inferred this from the grafana logs. However, i replaced the text panels with  [HTML plugin](https://grafana.com/grafana/plugins/aidanmountford-html-panel) and it works fine for me in v7.1.1

Closing this issue as it is not related to this reporter service.

Thanks

**Comment by Konradrundfunk** [comment]
I have the same problem but I am on version 6.4.4 what have you changed in order to function?

**Comment by dattasoumya92** [comment]
> I have the same problem but I am on version 6.4.4 what have you changed in order to function?

hi,

It worked for me in version 6.2.5 just fine. But when i upgraded to v 7.1.3 it failed rendering text panel, so I changed the Text visualization with this [HTML plugin](https://grafana.com/grafana/plugins/aidanmountford-html-panel) and it worked fine. I installed this plugin using grafana-cli in v7.1.3 and copied the content of text panels to html.

This issue is related to Direct Link Rendered Image feature in sharing option of Panels. I noticed that rendering image of text panels worked fine till v7.0.6. Since its a Grafana issue, hence i raised this bug in Grafana [link](https://github.com/grafana/grafana/issues/26996).


Thanks
